### ISSUE TITLE
purpose/docs of --user option for verdi computer configure/test

### ISSUE THREAD

**Issue by ltalirz** [aiidateam/aiida-core#3288]
`verdi computer configure ssh` has both a  `--user` and a `--username` option.
While the purpose of the `--username` option is clear, the purpose of the `--user` option is less so. Currently the help string reads:
```
verdi computer configure ssh -h
...
  -u, --user USER                 Email address of the user.
...
```

I get that this is an AiiDA user but what exactly does this user determine?
Does it determine which user the calculations are associated with when running on this computer? I guess not.

In other words: what is the use case where someone would want to specify a value for the `--user` option and configure a computer for a different user than the current user?

I think this just needs a little bit more explanation in the docstring. Happy to make a small PR to improve the help text once this is clear.

**Comment by sphuber** [comment]
The docstring is definitely very unclear,  but the concept makes a lot of sense I think. A `Computer` only defines how to connect to it, but it lacks authentication info. This is stored in an `AuthInfo` which has a ManyToOne relation with `Computer` and a OneToOne with a `User`. This means effectively that a single `Computer` can be used by many `Users` through individual `AuthInfos`. "Configuring" a computer then means to generate an authentication info for a given user. That is why the `verdi computer configure` command takes a `--user` option in case you want to configure the machine for a user other than the current default.
